src/test/java/org/broadinstitute/hellbender/MainTest.java

```diff
@@ -1,13 +1,53 @@
 package org.broadinstitute.hellbender;
 
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.TestProgramGroup;
 import org.broadinstitute.hellbender.exceptions.UserException;
-import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.junit.Assert;
 import org.testng.annotations.Test;
 
-public final class MainTest extends CommandLineProgramTest{
+import java.util.Collections;
+import java.util.List;
+
+public final class MainTest extends CommandLineProgramTest {
 
     @Test(expectedExceptions = UserException.class)
     public void testCommandNotFoundThrows(){
         this.runCommandLine(new String[]{"Brain"});
     }
+
+    @CommandLineProgramProperties(
+            programGroup = TestProgramGroup.class,
+            summary = "OmitFromCommanLine test",
+            oneLineSummary = "OmitFromCommanLine test",
```

Comment:
These two instances are still spelled wrong.

Reply:
Sorry, I don't know what I did to miss this two... Done